File: BookRent/Model/Person.cs
using System;

namespace BookRent
{
    /// <summary>
    /// 会员
    /// </summary>
    [Serializable]
    public class Person : ModelBase
    {
        public string Name { get; set; }
        public Sex Sex { get; set; }
        /// <summary>
        /// 开始日期
        /// </summary>
        public DateTime StartDate { get; set; }
        /// <summary>
        /// 到期日期
        /// </summary>
        public DateTime EndDate { get; set; }
        /// <summary>
        /// 会费
        /// </summary>
        public double Fee { get; set; }
        /// <summary>
        /// 押金
        /// </summary>
        public double Deposit { get; set; }
        /// <summary>
        /// 联系方式
        /// </summary>
        public string PhoneNo { get; set; }
        /// <summary>
        /// 拼音首字母
        /// </summary>
        public string Pinyin { get; set; }
        /// <summary>
        /// 联系人
        /// </summary>
        public string Contacter { get; set; }
        /// <summary>
        /// 备注
        /// </summary>
        public string Remark { get; set; }

        public override string ToString()
        {
            return string.Format("{0} {1}", Pinyin, Name);
        }
    }

    public enum Sex
    {
        男 = 0,
        女 = 1
    }
}

File: BookRent.Web/server/person.js
var _ = require('underscore')
var fake = require('../util/data_generator');
var persons = [];

module.exports = function(app){
    app.get('/person/list', function(req, res){
        res.end(JSON.stringify(persons));
    });
    app.get('/person/:id', function(req, res){
        var id = req.params.id;
        var person = _.findWhere(persons, {id: parseInt(id)});
        res.end(JSON.stringify(person));
    });
    app.post('/person/save', function(req, res){
        var person = req.body;
        if(null == person) return;

        if(null == person.id){
            person.id = persons.length;
            persons.push(person);
        }else{
            var index = _.findIndex(persons, {id: person.id});
            if(-1 != index){
                persons[index] = person;
            }
        }
        res.end(person.id.toString());
    });
    app.delete('/person/:id', function(req, res){
        console.log('delete person: id = '+req.params.id);
        var index = _.findIndex(persons, {id: parseInt(req.params.id)});
        if(-1 != index){
            persons.splice(index, 1);
        }
        res.end(req.params.id);
    });
    initPersons();
};

function initPersons(){
    console.log('init persons');
    for(var i=0;i<10;i++){
        persons.push({
            id:i,
            name:'alex',
            pinyin:'alex',
            sex:i%2,
            startDate:'2017-01-07',
            endDate:'2017-02-07',
            fee:100,
            deposit:200,
            phoneNo:'13700001111',
            contacter:'xiong',
            remark:'test'
        });
    }
}
